test/functional/wallet_silentpayments_receiving.py

```diff
@@ -147,7 +147,30 @@ def do_nothing():
 
         self.log.info("Wallet persistence verified successfully")
 
-    def test_import_rescan(self):
+    def test_import_descriptors(self):
+        self.log.info("Check import silent payments descriptor into blank wallet")
+
+        self.nodes[0].createwallet(wallet_name="import_blank", blank=True, silent_payments=True)
+        wallet = self.nodes[0].get_wallet_rpc("import_blank")
+
+        descriptor = "sp([eea23daf/352h/0h/0h/1h/0]cRCGWnoELVHTr9oZWz1TUp7jmdBJPS3Kx8UCaHvfpxL6KiVSHH1A,[eea23daf/352h/0h/0h/0h/0]cUxxbQ67tepsEn3AUKSCWAKrvLwP6MPk55DAptcqxqDiXN6yKkNW)#q42ue5dg"
+        res = wallet.importdescriptors([{
+            "desc": descriptor,
+            "active": True,
+            "next_index": 0,
```

Comment:
5be304dfa50bcfd3b0a92eff52bac3d8023ce523 nit: no need to set `next_index` during import

Reply:
Done